### PR TITLE
Adding CI build

### PR DESCRIPTION
**Pull request by conniey**
Adding CI build so that we can test PRs.

Fixes #352 

/cc @mmitche

### CODE REVIEW

**Comment by conniey** [comment]
test ci please

**Comment by conniey** [comment]
@mmitche I couldn't find an example of adding test results with `*.trx` results. Could you make sure its correct in my netci.groovy.  Also, am I missing a step? I don't see any updates after I added the `test ci please` comment.

**Comment by mmitche** [comment]
@conniey What format is .trx?

**Comment by mmitche** [comment]
@conniey Need to register the repo.  One sec.

File: netci.groovy [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Import the utility functionality.
+import jobs.generation.Utilities;
+
+def project = 'microsoft/dotnet-apiport'
+def branch = 'master'
```
Comment (mmitche) [inline netci.groovy]: These two lines should pull from incoming parameters:

def project = GithubProject
def branch = GithubBranchName

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by conniey** [comment]
> What format is .trx?

@mmitche .trx (Visual Studio Test Result) is a XML file

**Comment by conniey** [comment]
@mmitche Is there anything that I need to do to get this working?

**Comment by mmitche** [comment]
You check it in :)

**Comment by mmitche** [comment]
(as of right now the definitions in this file don't make it into this PR)

**Comment by conniey** [comment]
> You check it in :)

Oops... The obvious last step. :D

**Pull request merged**